frontend/views/components/BannerSimple.vue

```diff
@@ -20,14 +20,15 @@ export default {
   },
   validator: function (value) {
     // The value must match one of these strings
-    return ['info', 'warning', 'danger'].indexOf(value) !== -1
+    return ['success', 'info', 'warning', 'danger'].indexOf(value) !== -1
   },
   computed: {
     getIcon () {
       return {
-        warning: 'icon-exclamation-triangle',
-        danger: 'icon-times-circle',
-        info: 'icon-info-circle'
+        warning: 'icon-exclamation-triangle c-icon',
+        danger: 'icon-times-circle c-icon',
```

Comment:
But perhaps it's right to have the "X" here when it's used a "banner-simple" on its own (as in Group Rules), since there no additional "X" is added... so it seems almost like a conflict between this file and the other one above

Reply:
@mmbotelho can you review this?

Reply:
I would leave it as it is. It is common across the web, it's ubiquitous and also consistent with other close buttons we have in GI. And besides that, closing the banner is not a critical feature - meaning that it will not prevent or make an action harder to complete.